packages/portal/swagger-ui-web-component/package.json

```diff
@@ -41,7 +41,7 @@
     "raw-loader": "^4.0.2",
     "style-loader": "^3.3.4",
     "terser-webpack-plugin": "^5.3.10",
-    "webpack": "^5.95.0",
+    "webpack": "^5.101.0",
```

Comment:
I'm unsure if we should worry about updates in this package TBH, but no harm hopefully?